mod-di-converter-storage-server/src/main/java/org/folio/services/JobProfileServiceImpl.java

```diff
@@ -322,6 +326,25 @@ private static void validateAddedUpdateActionProfileAssociation(ProfileAssociati
     }
   }
 
+  private static void validateAddedUpdateActionProfileAssociationUnderSameMatch(List<ProfileAssociation> actionProfileAssociations,
+                                                                                List<ActionProfile> actionProfiles,
+                                                                                List<Error> errors) {
+    var updateActionProfiles = getUpdateProfileAssociations(actionProfileAssociations, actionProfiles);
+
+    if (!updateActionProfiles.isEmpty()) {
+      Map<ReactToType, Long> counts = updateActionProfiles.stream()
+        .collect(Collectors.groupingBy(ProfileAssociation::getReactTo, Collectors.counting()));
+
+      boolean hasMultipleActionsUnderSameMatch = counts.getOrDefault(ReactToType.MATCH, 0L) >= 2
+        || counts.getOrDefault(ReactToType.NON_MATCH, 0L) >= 2;
```

Comment:
@JavokhirAbdullayev  Let's move this value "2" to the separate constant.

Reply:
this value identifies whether one profile tree contains more than 1 update action (marc or instance), I found that making this constant is redundant and a bit misleading